modules/vactory_faq/vactory_faq.module

```diff
@@ -58,3 +58,41 @@ function vactory_faq_theme_suggestions_alter(array &$suggestions, array $variabl
     $suggestions[] = 'vactory_faq_accordion';
   }
 }
+
+
+/**
+ * Implements hook_json_api_collection_alter().
+ */
+function vactory_faq_json_api_collection_alter(&$filters, &$context) {
+  if ($context['id'] === 'vactory_faq_list') {
+    $query = \Drupal::request()->query->get("q");
+    if (empty($query)) {
+      return;
+    }
+
+    if (isset($query["page"])) {
+      $filters["page[offset]"] = intval($query["page"]) > 0 ? (intval($query["page"]) - 1) * $filters["page[limit]"] : 0;
+    }
+
+    if (isset($query["term"]) && $query["term"] !== "all") {
```

Comment:
@mkoumila 
Ajouter à la condition `&& !empty($query["term"])`
Dans le cas où `$query["term"] = ""` unne exception SQL sera généré utilisation de l'operateur IN sur empty list "`... IN ()`"